server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -227,8 +227,12 @@ class GdbDebugger : IDebugger {
             return field;
         }
          
-        public async Task<SSymbolResult> LookupGlobalSymbol(string module, string symbol, string typename) {
-            string pythonResult = await this.QueryDebuggerPython(String.Format("LookupGlobalSymbol(\"{0}\",\"{1}\")", module, symbol));
+        public async Task<SSymbolResult> LookupGlobalSymbol(string module, string symbol, string typename, string nameSpace) {
+            string symbolName = "";
+            if (!String.IsNullOrEmpty(nameSpace))
+                symbolName = nameSpace + "::";
+            symbolName += symbol;
+            string pythonResult = await this.QueryDebuggerPython(String.Format("LookupGlobalSymbol(\"{0}\",\"{1}\")", module, symbolName));

```

Comment:
When a type name is passed in, do we need to do something more here to find the right symbol?

Reply:
I don't know how to get a list of all global symbols with a matching name in gdb... it usually disambiguates by filename, I think. https://sourceware.org/gdb/onlinedocs/gdb/Symbols.html#Symbols

Was there a specific symbol that prompted the addition of this parameter?

Reply:
Yes, the AXTree extension looks for chrome!instance, which has many occurrences, so the type name parameter is used to specify which one it wants. If you want to try that extension, you’ll need to enable accessibility settings in chrome://accessibility (there’s also a flag you can pass to Chromium during launch to enable these, but I don’t remember what it’s called), and you’ll have to attach to the browser process.

Reply:
Hmm, I see. I'll have to figure out something. Is it OK to address that separately though?

Reply:
If that's easier, sure. Are you going to work on that immediately? Otherwise, might be worth leaving a TODO so we don't forget about it.

Reply:
OK, added a TODO.